From 22e1ad4307723df0f6b57a04e89a61c33b865264 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 14 May 2024 15:24:18 +0200
Subject: libstdc++: real time reporting bindings vector requires full
 declaration

libstdc++ implementation of std::vector requires the full declaration
of the type and not a forward declaration. This breaks compilation of
RealTimeReportingBindings, as it has an std::vector of
RealTimeReportingContributionPtr.

Bug: 41455655
Change-Id: I9ee70fbb58af36f4cfc43a5d4b1e99799f4c760c
---
 content/services/auction_worklet/real_time_reporting_bindings.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/content/services/auction_worklet/real_time_reporting_bindings.h b/content/services/auction_worklet/real_time_reporting_bindings.h
index c755d5cab041e..da4d05e96f788 100644
--- a/content/services/auction_worklet/real_time_reporting_bindings.h
+++ b/content/services/auction_worklet/real_time_reporting_bindings.h
@@ -12,7 +12,7 @@
 #include "content/common/content_export.h"
 #include "content/services/auction_worklet/auction_v8_helper.h"
 #include "content/services/auction_worklet/context_recycler.h"
-#include "content/services/auction_worklet/public/mojom/real_time_reporting.mojom-forward.h"
+#include "content/services/auction_worklet/public/mojom/real_time_reporting.mojom.h"
 #include "v8/include/v8-forward.h"
 
 namespace auction_worklet {
-- 
2.40.1

